### PR TITLE
[chaosplt 207] namespace tag gone

### PR DESCRIPTION
**Pull request by clairecng**
## What does this PR do?

- [ ] Adds new functionality
- [x] Alters existing functionality
- [ ] Fixes a bug
- [x] Improves documentation or testing

Please briefly describe your changes as well as the motivation behind them:
- Finally remove all of the bad namespaces tags
- removes instance tag
- move every recurrent tag to use consts from log pkg

Our logs were using multiple different tags for the same resource.

## Code Quality Checklist

- [ ] The documentation is up to date.
- [ ] My code is sufficiently commented and passes continuous integration checks.
- [ ] I have signed my commit (see [Contributing Docs](../CONTRIBUTING.md)).

## Testing

- [ ] I leveraged continuous integration testing
    - [ ] by depending on existing `unit` tests or `end-to-end` tests.
    - [ ] by adding new `unit` tests or `end-to-end` tests.
- [ ] I manually tested the following steps:
    - `x`
    - [ ] locally.
    - [ ] as a canary deployment to a cluster.

### CODE REVIEW

File: api/v1beta1/disruption_webhook.go [hunk 1]
```diff
@@ -113,7 +114,7 @@ var _ webhook.Defaulter = &Disruption{}
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (r *Disruption) Default() {
 	if r.Spec.Duration.Duration() == 0 {
-		logger.Infow(fmt.Sprintf("setting default duration of %s in disruption", defaultDuration), "instance", r.Name, "namespace", r.Namespace)
+		logger.Infow(fmt.Sprintf("setting default duration of %s in disruption", defaultDuration), cLog.DisruptionNameKey, r.Name, cLog.DisruptionNamespaceKey, r.Namespace)
```
Comment (ptnapoleon) [inline api/v1beta1/disruption_webhook.go:117]: this is gonna disappear on rebase

**Review by ptnapoleon — APPROVED** [review]

**Review by ptnapoleon — APPROVED** [review]

**Review by aymericDD — APPROVED** [review]
🧽

File: controllers/disruption_cron_controller.go [hunk 2]
```diff
@@ -35,7 +36,7 @@ type DisruptionCronReconciler struct {
 }
 
 func (r *DisruptionCronReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ctrl.Result, err error) {
-	r.log = r.BaseLog.With("disruptionCronNamespace", req.Namespace, "disruptionCronName", req.Name)
+	r.log = r.BaseLog.With(cLog.DisruptionCronNamespaceKey, req.Namespace, cLog.DisruptionCronNamespaceKey, req.Name)
```
Comment (clairecng) [inline controllers/disruption_cron_controller.go:39]:
```suggestion
	r.log = r.BaseLog.With(cLog.DisruptionCronNamespaceKey, req.Namespace, cLog.DisruptionCronNameKey, req.Name)
```

**Pull request merged**